components/dev-tools/easybuild/SPECS/easybuild.spec

```diff
@@ -48,25 +48,24 @@ you to manage (scientific) software on High Performance Computing (HPC)
 systems in an efficient way.
 
 %prep
-mkdir %{buildroot}
+%setup -q -n %{pname}-%{version} -a1 -a2 -a3
 
 %build
-
-cd %{buildroot}
-cp %{_sourcedir}/*py .
-
-export EASYBUILD_BOOTSTRAP_SOURCEPATH=%{_sourcedir}
-export EASYBUILD_INSTALLPATH=%{install_path}
-export EASYBUILD_MODULE_SYNTAX=Tcl
-export PATH=${LMOD_DIR}:${PATH}
-
-MODULEPATH= python3 ./bootstrap_eb.py %{buildroot}/%{install_path}
-
-rm bootstrap_eb.py*
-pushd %{buildroot}%{install_path}/modules/tools/EasyBuild/
-rm %version
-ln -s ../../all/EasyBuild/%version .
-popd
+%{__python3} setup.py build
+
+for eb in easyblocks easyconfigs framework; do
+	cd %{pname}-${eb}-%{version}
+	%{__python3} setup.py build
```

Comment:
This can be skipped, it doesn't do anything at all in the case of EasyBuild

Reply:
skipped